### PR TITLE
Extract features for OSM multiyear and Google Buildings v3

### PR DESCRIPTION
**Pull request by tm-abby-moreno**
## What does this PR do?

Two extractions were done in this PR: OSM (multiple years) and re-extraction of Google Buildings v3.

- **OSM multiple years:** POIs, water, and waterway features were extracted for each OSM year. Years available are from 2014-2022.
- **Google Buildings v3:** Added the creation of different features (building count, building density, percentage of buildings for each size threshold, etc.).   
  - There was supposed to be a separate notebook for cropping the google buildings v3, however, it was in my old device which broke down. Only able to recreate the features notebook.

## Where should the reviewers start?

- `notebooks` > `tm` > `osm` > `03-extract-multiyear-osm-features.ipynb`
- `notebooks` > `tm` > `building_footprints` > `4_extract_features.ipynb`

### CODE REVIEW

File: notebooks/tm/building_footprints/4_extract_features.ipynb [hunk 1]
```diff
@@ -0,0 +1,851 @@
+{
```
Comment (tm-jace-peralta) [inline notebooks/tm/building_footprints/4_extract_features.ipynb:1]: <p>All columns containing count of bldgs must be integers, kindly convert</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (tm-jace-peralta) [inline notebooks/tm/building_footprints/4_extract_features.ipynb:1]: <p>For consistency, revise to:</p><p>    "ADM4_PCODE",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"brgy_area",</p><p>    "bldg_count",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_total_area",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_mean_area",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_lt100sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_100_200sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_count_gt200sqm",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_area_density",</p><p>&nbsp;&nbsp;&nbsp;&nbsp;"bldg_pct_brgy_area"</p><p> </p><p>Kindly apply this also in the column parameter in the <code>vzs.create_zonal_stats</code> function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

File: notebooks/tm/osm/04-compare-osm-years.ipynb [hunk 2]
```diff
@@ -0,0 +1,1512 @@
+{
```
Comment (tm-jace-peralta) [inline notebooks/tm/osm/04-compare-osm-years.ipynb:1]: <p>Is this still needed? can delete if not used</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (tm-jace-peralta) [inline notebooks/tm/osm/04-compare-osm-years.ipynb:1]: <p>Is this still needed? can delete if not used. but if relevant, run to show output</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/thinkingmachines/lacuna-fund-climate-health/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

File: notebooks/tm/building_footprints/4_extract_features.ipynb (hunk 1 above)
Comment (tm-abby-moreno) [reply notebooks/tm/building_footprints/4_extract_features.ipynb:1]: <p>Noting that will rename columns according to recent ERD. </p>

File: notebooks/tm/osm/04-compare-osm-years.ipynb (hunk 2 above)
Comment (tm-abby-moreno) [reply notebooks/tm/osm/04-compare-osm-years.ipynb]: <p>Ran to show output and added a markdown for clarity.</p>

**Comment by tm-abby-moreno** [comment]
Done revising comments here! for column names, aligned them to ERD. Thanks!

**Review by tm-jace-peralta — APPROVED** [review]

**Pull request merged**